tests/DoctrineManagerTest.php

```diff
@@ -1,19 +1,21 @@
 <?php
 
+namespace LaravelDoctrine\Tests;
+
 use Doctrine\Common\EventManager;
 use Doctrine\Common\Persistence\ManagerRegistry;
 use Doctrine\DBAL\Connection;
 use Doctrine\ORM\Configuration;
 use Doctrine\ORM\EntityManagerInterface;
 use Illuminate\Contracts\Container\Container;
+use InvalidArgumentException;
```

Comment:
I don't think importing from global namespace is needed.  If we really need to specify that it is from global namespace we can use the fully qualified name when using the name.  It is actually a small optimization to do it like that anyway.

e.g. `\InvalidArgumentException::class`